MANIFEST.in

```diff
@@ -0,0 +1,27 @@
+# Manifest template
```

Comment:
@sashafrey It is for specifying which files are to include in source distribution.